aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -643,7 +648,7 @@ Resources:
           #!/bin/bash -x
           <%=indent(erb_file(BOOTSTRAP_CHEF,
             resource_id: daemon,
-            node_name: frontends ? '$ENVIRONMENT-daemon' : '$STACK',
+            node_name: frontends ? '$STACK-daemon' : '$STACK',
```

Comment:
Minor nit: Could we rename `$STACK` to `$STACKNAME` to align with the cloudformation template attribute?